CHANGES.rst

```diff
@@ -108,6 +108,8 @@ Bug Fixes
 - Prevent duplicate labels by changing duplicate number appended
   to label to max number (of duplicates) plus 1. [#1824]
 
+- Layer lettering now supports up to 700 layers. [#1850]
```

Comment:
Would this be clearer?
```suggestion
- Layer lettering now supports up to 702 layers. Beyond that, special characters are used. [#1850]
```

Reply:
sure, I'm not sure that users care about the difference, but is at least good bookkeeping for ourselves